service/matching/fx.go

```diff
@@ -55,7 +55,7 @@ var Module = fx.Options(
 	fx.Provide(RateLimitInterceptorProvider),
 	fx.Provide(HandlerProvider),
 	fx.Provide(service.GrpcServerOptionsProvider),
-	fx.Provide(TaskQueueReplicatorNamespaceReplicationQueueProvider),
+	fx.Provide(NamespaceReplicationQueueProvider),
```

Comment:
I wasn't objecting to the long name, I just thought it was funny. either way is fine

Reply:
Oh, in any case I think the shorter name is a bit nicer.